tests/test_aligner.yml

```diff
@@ -37,15 +49,18 @@
     - dragmap
     - preprocessing
   files:
-    - path: results/multiqc
+    - path: chr22_1-40001.bed.bed.gz
```

Comment:
bed.bed seems wrong. maybe something changed with the bgzip modules

Reply:
or bad copy paste from my side as it's one of the checks that fails...

Reply:
Actually both, but now fixed